backend/app/routes/index.js

```diff
@@ -5,11 +5,15 @@ const { emailTest } = require('./testRoutes/emailTest');
 const tinyURL = require('./tinyURL');
 const broadcast = require('./broadcast');
 const FAQ = require('./faq/post');
+const GetFaq = require('./faq/GetFaq');
+const DeleteFaq = require('./faq/DeleteFaq');
```

Comment:
Please replace above line with-
`const deleteFaq = require('./faq/delete-faq');`

and make the required change in `faq/index.js` file